code-editor/scripts/on-start.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+set -ex
+ASI_VERSION=0.3.0
+
+# PARAMETERS 
+IDLE_TIME=3600  # in seconds, change this to desired idleness time before app shuts down
+
+AUTOSTOP_FILE_URL="https://github.com/aws-samples/sagemaker-studio-apps-lifecycle-config-examples/releases/download/sagemaker_code_editor_auto_shut_down-$ASI_VERSION.tar.gz"
```

Comment:
Missing release in the URI e.g. download/v$ASI_VERSION

Reply:
Updated